server/server.go

```diff
@@ -405,6 +409,16 @@ func (s *Server) StartExitMonitor() {
 							}
 						}
 					}
+					if c != nil && s.watcherWaitContainer(containerID) {
```

Comment:
We can add a check for the state == stopped here as well as that will be the case for runc and we can avoid spawning a goroutine.

Reply:
yes good point !

Reply:
done